src/common/runner/runner.go

```diff
@@ -57,7 +57,25 @@ func (r *Runner) Init(commands *clioptions.TagOptions) error {
 			externalTagGroup.InitExternalTagGroups(commands.ConfigFile)
 		}
 	}
-	r.parsers = append(r.parsers, &tfStructure.TerrraformParser{}, &cfnStructure.CloudformationParser{}, &slsStructure.ServerlessParser{})
+	for _, value := range commands.Parsers {
+		if value == "Terraform" {
+			r.parsers = append(r.parsers, &tfStructure.TerrraformParser{})
+			break
+		}
+	}
+	for _, value := range commands.Parsers {
+		if value == "CloudFormation" {
+			r.parsers = append(r.parsers, &cfnStructure.CloudformationParser{})
+			break
+		}
+	}
+	for _, value := range commands.Parsers {
+		if value == "Serverless" {
+			r.parsers = append(r.parsers, &slsStructure.ServerlessParser{})
+			break
+		}
+	}
```

Comment:
@alexberdan Please merge these for loops into a single one, with switch cases for the different values. Avoid duplicates by cleaning the `commands.Parsers` list in advance 🙏

Reply:
Thank you, is my new construction better?

Reply:
Well, it's erroring, but I don't know why 😬 Can you help?